### ISSUE TITLE
legendary spawning items

### ISSUE THREAD

**Issue by BigChungv2** [SmashMC-Development/Suggestions#984]
**Describe the suggestion**
due to certain legendary spawns being so rare or non existant in warzone, could we add the items to spawn such legendaries like the clear and tidal bells to the black market shop, however have them cost a high price like 150/200k, so people dont just abuse the items easily

**Which servers does this apply to?**
all 

**Additional context**
n/a

**Comment by Firemelon66** [comment]
I like this since u like sorta not really able to get tidal/clear bells komoonium z ultranecrozmium z and i mean the prices could just be quite high so people have to decide for them self if they wanna hunt the item or just buy it from the server so like megastones and so on

**Comment by GiffyG** [comment]
I disagree with the addition of these items in the shop, since it would ruin the economy of searching for these items in the first place. Since there are those who do not do Warzone as a method of enjoying the game, but rather collect and resell rare items that are farmed for, and by just selling them at the shop, it ruins that aspect of gameplay for collectors. Another reason is because it would seem highly unbalanced, seeing as of currently it’s a 5% chance of wild drops from legendaries/mons for specific Z-crystals, for a good reason that if there’s too many of those rare held items, i.e there’d be a bunch of people running around with Ultra Necrozma in Warzone, it would push back new players from entering those gamemodes for those simply with a lot of money in order to afford these from the shop, further opening the gap between rich players and starters. Having these items be obtained only through farming is the limiter needed to help balance it more to those who dedicated themselves to it and those with just money to spend.

**Comment by morrow12** [comment]
Yes they should add the legendary mega stones for like 250k and the stones for specific non leg mons for 80k

**Comment by BigChungv2** [comment]
I don’t specifically mean Z crystals like ultra nec Z, I can understand not adding them as you have the 5% chance when defeating a necrozma to get it, but however things like tidal bell or clear bell maybe should be added, we’re what 6 months past the reset? I haven’t seen a single person with a clear bell on kyurem and maybe 2 ho-ohs on the server, yeah they’ve now been added to the event crate so they’re now available to get but they’re going to be still incredibly rare to get
